MAUI/Avatar-view/customization.md

```diff
@@ -466,3 +466,30 @@ this.Content = mainGrid;
 
 ![SfAvatarView content padding](Customization_Images/ContentPadding_AvatarView.png)
 
+### Font auto scaling enabled
+
+The FontAutoScalingEnabled property is used to automatically scale the avatar view's font size based on the operating system's text size. The default value of the `FontAutoScalingEnabled` property is `false.`
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<Grid>
+   <sfavatar:SfAvatarView  FontAutoScalingEnabled = true>
```

Comment:
Please check the code snippet

Reply:
I have checked and updated the code snippet.